lambda-loader/src/data.py

```diff
@@ -61,7 +61,8 @@ def create_support_table(self, table_name, key):
 
     def parse_time(self, time_str):
         # the -3 converts from nanoseconds to microseconds which the datetime class can handle
-        time = datetime.strptime(time_str[:-3], '%Y-%m-%d %H:%M:%S.%f')
+        time_str = time_str[:-3] + ' UTC'
```

Comment:
If UTC is added to the time string here, does this interfere with the comparison of the time string obtained from the leaderboard api?

Reply:
`'utc'` is specified in the time string from the api, this just adds it back. maybe it should be a parameter.